File: api-examples/cat/search/java/spatial_search_polygon.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("https://catalogue.iudx.org.in/catalogue/v1/search?geometry=polygon%28%2818.4,73.9,21.6,78.9,27.1,80,30,75.25,25.7,74.7,18.4,73.9%29%29&relation=within")
  .get()
  .build();

Response response = client.newCall(request).execute();

File: api-examples/cat/count/Go/spatial_search_polygon.go
package main

import (
  "fmt"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://<catalogue-server-ip>/catalogue/v1/count?geometry=polygon((18.4,73.9,21.6,78.9,27.1,80,30,75.25,25.7,74.7,18.4,73.9))&relation=within"
  method := "GET"

  client := &http.Client {
  }
  req, err := http.NewRequest(method, url, nil)

  if err != nil {
    fmt.Println(err)
  }
  res, err := client.Do(req)
  defer res.Body.Close()
  body, err := ioutil.ReadAll(res.Body)

  fmt.Println(string(body))
}
